syntax = "proto3";

package studio.magicmedia.vacancy.api.v1;

option go_package = "cyrex/vacancies/pb";
import "studio/magicmedia/api/v1/vacancy.proto";

message CreateVacancyRequest {
  string Title = 1;
  string Description = 2;
  Vacancy.DIVISION Division = 3;
  string Country = 4;
}
